blog/2014/dev-portal-adds-rippled-apis.md

```diff
@@ -1,7 +1,16 @@
+---
+category: 2014
+date: 2014-08-01
+labels:
+    - Development
+theme:
+    markdown:
+        editPage:
+            hide: true
+author: Rome Reginelli
```

Comment:
We could consider enhancing the contributor information in the near future - maybe replacing names with icons and linking to the author's GitHub or LinkedIn profile.

Reply:
Before we go that far, I'd like to improve the template for the individual blog post pages so it shows the author and original post date. Right now it doesn't show either one, just the last edit time.